plugins/inputs/vsphere/README.md

```diff
@@ -64,12 +69,17 @@ vm_metric_exclude = [ "*" ]
     "virtualDisk.writeOIO.latest",
     "sys.uptime.latest",
   ]
-  # vm_metric_exclude = [] ## Nothing is excluded by default
-  # vm_instances = true ## true by default
+  vm_metric_exclude = []
 
-  ## Hosts
-  ## Typical host metrics (if omitted or empty, all metrics are collected)
-  # host_include = [ "/*/host/**"] # Inventory path to hosts to collect (by default all are collected)
+  ## If true, gather virtual machine instance metrics.
+  # vm_instances = true
+
+  ## Inventory path of the hosts to gather metrics on.  If empty, all hosts
+  ## are included.
+  ##   ex: host_include = [ "/*/host/**"]
```

Comment:
Make this path "/**" instead. Gets rid of that permission issue someone ran into a while ago. Requires a change to the default struct initializer in vsphere.go. See #5613